core/src/main/res/drawable/ic_x_custom_off.xml

```diff
@@ -0,0 +1,19 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Specifying them here in vector format should be enough. You do not need to add the png files. Instead of having a `on` and `off` version, please use `android:alpha` on the views. This way, you need fewer different drawables. Having to support each version for each theme will otherwise require a ton of drawables.